### ISSUE TITLE
Instagram landscape video

### ISSUE THREAD

**Issue by gianpaj** [jjahr/ffmpeg-droplet#2]
https://dustn.tv/instagram-sizes/

> Landscape videos [..] have a maximum display size of 600×336.5 (or 1200×673 in full view or retina display). This is very close to a typical 16:9 ratio that we see as the current cinematic standard in film & television.

**Comment by jjahr** [comment]
That's true, thanks for the reminder.  To make the droplet more future proof I'll add a custom display size so users can type in their own size if they choose to.
